public/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
Rename the title to `Solid Web app with React`.

Reply:
Changed the title in [commit 7dbfeb8](https://github.com/SolidLabResearch/solid-web-app-react-template/pull/1/commits/7dbfeb829676d99a6191f8f703a734255d0ebad2)